recipes/xtb-python/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "xtb-python" %}
+{% set version = "20.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/grimme-lab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: fc0cfad603324c4a41c575c7c3bf671bf68f03b1b9f0e40e8dc501a56b1b9b1e
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - meson >=0.51
+    - ninja
+    - python
+    - cffi
+  host:
+    - python
+    - pip
+    - cffi
+    - xtb 6.3.*
```

Do you need to specify the version here? Or can you rely on the latest instead?
```suggestion
    - xtb
```

---

There is a 6.2 version available which does not provide the required C-API, maybe `>=6.3` ist the better choice.

---

`>=6.3` is okay provided `xtb` is never included in the global pinnings. But as this provides a shared library I'd suggest not having a version specified and instead relying on the fact that conda will automatically pick the newest for the build. This constraint will be propagated to installations due to the `pin_compatible` so it won't be possible to install this with `xtb==6.2`.